package/src/routing/register.ts

```diff
@@ -76,7 +76,9 @@ const generateRoute = (
 
 		let content = readFileSync(page.entrypoint, "utf-8");
 
-		content = content.replaceAll("getLocalePlaceholder()", `"${locale}"`);
+		content = content
+			.replaceAll("getLocalePlaceholder()", `"${locale}"`)
+			.replaceAll("getDefaultLocalePlaceholder()", `"${defaultLocale}"`);
```

Comment:
Ah interesting thank you! Indeed it's the same no matter the page. I guess globalThis doesn't cause issues in this case because the value won't change between requests right?

Reply:
Exactly

Reply:
For the sake of simplicity, I'll keep it as is for now but I keep it in mind!